### DISCUSSION TITLE
Palworld

### DISCUSSION THREAD

**Discussion by ssiwm** [gameserverapp/Platform#716] [Game request]
premiere scheduled for January 19, 2024

steam: https://store.steampowered.com/app/1623730/Palworld/

servers: expected on launch day

![obraz](https://github.com/gameserverapp/Platform/assets/105603502/c67d78df-a0fd-4fa6-bf7e-06652f2e005a)
source: https://discord.gg/pocketpair

update: servers https://tech.palworldgame.com/dedicated-server-guide

**Comment by kaiotterbark** [comment]
was just looking into if GSA was going to support this, i would use for sure

**Comment by scarecr0w12** [comment]
Easy setup for both Linux and windows, Game runs nice, Rcon is available in a simplistic form, and settings are similar to ASA

**Comment by Spencer021** [comment]
with the current player count 355K. This would be a great addition to GSA

**Comment by mvaessen** [comment]
Have spent a couple hours today trying to get it to work. So far it's crashing on startup on Linux and not even starting on Windows in docker.

Will take another look somewhere next week.

For people that want to help: try get it running with one of the GSA docker images, or create a new one using the `gameserverapp-base` docker image.

**Reply by scarecr0w12** [reply]
Can you provide the output you got, all of it? i just want to see what its showing for you when you think its crashing, to compare what im seeing from pterodactyl

**Comment by ad3d0** [comment]
I've noticed one thing, running it on windows requires DirectX Runtime installed, which is quite unusual for dedicated servers. May be that's why it was not starting for you at all on Windows,
And a side note for when you got it working, it requires `-EpicApp=PalServer` startup parameter for server to be visible in server list.

**Reply by Mrtritolo** [reply]
Also other game manager use directx for palworld

**Comment by BlackDahlia313** [comment]
support has been added for linuxgsm fyi if wanted to add rcon support to start :)

**Comment by mvaessen** [comment]
Palworld is now available for Windows & Linux!
